scripts/build_test.py

```diff
@@ -83,22 +86,6 @@ def test_minify_and_create_sourcemap(self):
         # `returncode` is the exit status of the child process.
         self.assertEqual(called_process.exception.returncode, 1)
 
-    def test_ensure_files_exist(self):
-        """Test _ensure_files_exist raises exception with a non-existent
-        filepath.
-        """
-        non_existent_filepaths = [INVALID_INPUT_FILEPATH]
-        # Escape the special characters, like '\', in the file paths.
-        # The '\' character is usually seem in Windows style path.
-        # https://docs.python.org/2/library/os.html#os.sep
-        # https://docs.python.org/2/library/re.html#regular-expression-syntax
-        error_message = ('File %s does not exist.') % re.escape(
-            non_existent_filepaths[0])
-        # Exception will be raised at first file determined to be non-existent.
-        with self.assertRaisesRegex(
-            OSError, error_message):
-            build._ensure_files_exist(non_existent_filepaths)  # pylint: disable=protected-access
-
```

Comment:
I only see the deletion of this test, is there any test written around the same behavior?

Reply:
Yes, I have added the similar behavior test [here](https://github.com/oppia/oppia/pull/15020/files#diff-dde3004aa3914bfc4d27dc6220ff5bc5273c411113c277a0a13e25861815d4e0R849) in this PR only.